src/rgw/driver/rados/rgw_bucket.cc

```diff
@@ -28,6 +28,13 @@ constexpr uint64_t BUCKET_TAG_QUICK_TIMEOUT = 30;
 
 using namespace std;
 
+static const string BI_OLH_ENTRY_NS_START = "\x80" "1001_";
+static const string BI_INSTANCE_ENTRY_NS_START = "\x80" "1000_";
```

Comment:
could you please add a comment saying these values are copied from cls/rgw/cls_rgw.cc?